maintenance-branch

```diff
@@ -1 +1 @@
-4-0-dev
\ No newline at end of file
+drop-old-rubies
```

Comment:
⚠️ This change is temporary and needs to be removed before merging, or soon after ⚠️

Reply:
Same as the others I don't get why these are interdependant apart from rspec-support which will be merged last.

Reply:
Reverted 👍